packages/react-ui-core/src/List/ListItem.js

```diff
@@ -0,0 +1,45 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import cn from 'classnames'
+import themed from 'react-themed'
+
+@themed('*', { pure: true })
+
+export default class ListItem extends PureComponent {
+
+  static propTypes = {
+    theme: PropTypes.object,
+    className: PropTypes.string,
+    orientation: PropTypes.string,
+    listItemNodeType: PropTypes.string,
```

Comment:
Let's rename this to just `nodeType`. Fixing what I suggest [above](https://github.com/rentpath/react-ui/pull/75/files#diff-955b1a65301832dda07a85d377a06f87R35) should rectify this prop name.